akka-sample-cluster-java/build.sbt

```diff
@@ -1,7 +1,10 @@
 import com.typesafe.sbt.SbtMultiJvm.multiJvmSettings
 import com.typesafe.sbt.SbtMultiJvm.MultiJvmKeys.MultiJvm
 
-val akkaVersion = "2.7.0"
+val AkkaVersion = "2.7.0"
+val AkkaDiagnosticsVersion = "2.0.0-M3"
+val LogbackClassicVersion = "1.2.11" 
+val ScalaTestVersion = "3.0.8"
```

Comment:
same thing in several places

Reply:
`"3.0.8"` was the previous version in the build.sbt (in most build.sbt). Are you suggesting bumping it to `3.1.0` in all?

Reply:
ok, while you are anyway changing it you can bump to 3.1.0

Reply:
There is one project with '3.1.1' I'll use that for the rest.